tests/plugins/colorbutton/manual/custompanelrender.md

```diff
@@ -0,0 +1,13 @@
+@bender-tags: 4.16.1, bug, 4247
+@bender-ui: collapsed 
+@bender-ckeditor-plugins: wysiwygarea, toolbar, colorbutton, colordialog, sourcearea
+
+1. Click **Text Color** or **Background Color** button.
```

Comment:
Current procedure suggests that the test can be done only for the one button. But it's better to test both of the buttons. So I propose to make the first step (with expected and unexpected sections) for Text Color button and then the second step like:

```markdown
2. Repeat the procedure for the **Background Color** button.
```